core/modules/czar/python/proxy.py

```diff
@@ -50,9 +63,11 @@ class Lock:
     requests when it is waiting on mysql queries that it knowingly
     initiates in its C-layer, whereas it treats its Lua plugin code as a
     single-threaded black-box."""
-    createTmpl = "CREATE TABLE IF NOT EXISTS %s (chunkId INT, code SMALLINT, message CHAR(255), timeStamp FLOAT) ENGINE=MEMORY;"
+    createTmpl = "CREATE TABLE IF NOT EXISTS %s " \
+                 "(chunkId INT, code SMALLINT, message CHAR(255), " \
```

Comment:
actually MEMORY tables can't contain TEXT. I think we just need to make sure we have a reasonable fixed size CHAR (and maybe check whether we are hitting the limit, and log it somewhere)